config/daemon/prometheus.md

```diff
@@ -184,7 +184,7 @@ scrape_configs:
     # scheme defaults to 'http'.
 
     static_configs:
-      - targets: ['docker.for.win.localhost:9090']
+      - targets: ['host.docker.internal:9090'] # Only works on Docker Desktop for Windows
```

Comment:
Are the "for Windows" and "for Mac" examples now the same? Wondering if we should remove one of them